src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
+    private String dueTime;
+
+    public Deadline(String taskInfo, String dueTime){
+        super(taskInfo);
+        this.dueTime = dueTime;
+    }
+
+    @Override
+    public String toString() {
+        String completion = " ";
+        if (super.isDone()){
```

Comment:
You can use Ctrl + Alt + L on IntelliJ to format your code! For standardised spacing, particularly "method() {"

Reply:
Cheers for the tip!